composer.json

```diff
@@ -9,8 +9,8 @@
         "php": ">=5.6,<8.0-dev",
         "ext-openssl": "*",
         "simplesamlphp/saml2": "3.2.*",
-        "symfony/dependency-injection": ">=2.7,<4",
-        "symfony/framework-bundle": ">=2.7,<4",
+        "symfony/dependency-injection": ">=2.7,<4 || ~4.3",
```

Comment:
Do you have a good reason why you would only allow SF 4.3 and up? 
Are you certain the bundle still works with Symfony 2.7? You could include that version in the test stages. But I think it's even better to simply drop support for that version. I know of no Stepup application that uses Symfony 2.7.